### PR TITLE
adding playbook to nova & cinder alerts

### PR DESCRIPTION
**Pull request by rajivmucheli**
@Carthaca 

Please validate if the rules & flow are correct, i am currently adding info to the documentation links.

### CODE REVIEW

**Comment by Carthaca** [comment]
Hi, when moving to a template you need to add the alerts here: https://github.com/sapcc/helm-charts/blob/master/system/kube-monitoring/charts/prometheus-frontend/templates/config.yaml

File: system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl [hunk 1]
```diff
@@ -23,7 +26,10 @@ ALERT OpenstackCinderVolumeAttach
     severity = "info",
     tier = "openstack",
     dashboard = "cinder",
-    meta = "{{ $labels.host }} has volumes stuck in attach state"
+    meta = "{{`{{ $labels.host }}`}} has volumes stuck in attach state"
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl]: comma missing

File: system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl [hunk 2]
```diff
@@ -38,7 +44,10 @@ ALERT OpenstackCinderVolumeDetach
     severity = "info",
     tier = "openstack",
     dashboard = "cinder",
-    meta = "{{ $labels.host }} has volumes stuck in detach state"
+    meta = "{{`{{ $labels.host }}`}} has volumes stuck in detach state"
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl]: comma missing

File: system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl [hunk 3]
```diff
@@ -23,7 +26,10 @@ ALERT OpenstackCinderVolumeAttach
     severity = "info",
     tier = "openstack",
     dashboard = "cinder",
-    meta = "{{ $labels.host }} has volumes stuck in attach state"
+    meta = "{{`{{ $labels.host }}`}} has volumes stuck in attach state"
+    {{ if .Values.ops_docu_url -}}
+      playbook = "{{.Values.ops_docu_url}}/docs/support/playbook/volumes.html",
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-cinder.alerts.tpl]: can you please make sure that the content of the playbook fits to the alert?

File: system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-nova.alerts.tpl [hunk 4]
```diff
@@ -87,7 +87,10 @@ ALERT OpenstackNovaInstanceStuckDeleting
     severity = "info",
     tier = "openstack",
     dashboard = "nova-hypervisor",
-    meta = "{{ $value }} instances"
+    meta = "{{`{{ $value }}`}}instances",
```
Comment (Carthaca) [inline system/kube-monitoring/charts/prometheus-frontend/templates/_openstack-nova.alerts.tpl]: space missing

**Comment by rajivmucheli** [comment]
Closing PR due to conflicts.

**Pull request closed without merging**